neo4j-examples/paradise-papers-django
# Docs: Part 4

Closes [Ticket](https://trello.com)

How to query the Neo4j database with Neomodel.

## Testing

Steps to manually verify the change:
1. _Setup_: cd docs/
2. _Run_: make html 
3. _Verify_: You can check the output of the above command in the _build/ directory
